share/cmake/utils/CheckSupportGL.cmake

```diff
@@ -35,6 +36,35 @@ if((OCIO_BUILD_TESTS AND OCIO_BUILD_GPU_TESTS) OR OCIO_BUILD_APPS)
                 set(GLEW_LIBRARIES GLEW::GLEW)
             endif()
         endif()
+
+        # Cmake has a bug with FindGLEW. It fails to defined GLEW_INCLUDE_DIRS in some situation.
```

Comment:
Question: Is the comment on line 33-34 still valid or should it be removed? It seems the CMake bug is still existing and not fixed as suggested on that comment.

Reply:
I'll remove it. It is not valid anymore as that issue was closed by accident and re-opened.